pkg/resiliency/breaker/circuitbreaker_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package breaker_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/pkg/expr"
+	"github.com/dapr/dapr/pkg/resiliency/breaker"
+)
+
+func TestCircuitBreaker(t *testing.T) {
```

Comment:
Yes, we do need more tests that exercise timeout, retry, and circuit breaking. My thought was this would come in a later PR. For timeouts, we will also need all component interfaces to accept `ctx context.Context`. (added this to the next steps in the description)